docs/basics/page.md

```diff
@@ -1,22 +1,49 @@
 (publish:page)=
-# Build a single page
+# Build an article
 
-Sometimes you'd like to build a single page of content rather than an entire book.
-For example, if you'd like to generate a web-friendly HTML page from a Jupyter notebook for a report or publication.
+You can build an **article** (e.g., a single page) rather than an entire book.
+You can build outputs for an article from a single source file, or split it up across multiple files (similar to how you'd structure a book).
 
-You can generate a standalone HTML file for a single page of the Jupyter Book using the same command:
+This section contains more information about how to do this.
+
+:::{admonition} Work in progress
+Article building functionality for Jupyter Book is still under design and development.
+This functionality may change over time!
+If you have ideas, suggestions, or would like to help out, please [see the contributing guide](../contribute/intro.md).
+:::
+
+## Build an article from a single file
+
+You can generate a standalone HTML file for a single page of the Jupyter Book using the `jupyter-book` command, and pointing it to a **single file** instead of a book's directory:
 
 ```bash
 jupyter-book build path/to/mypage.ipynb
 ```
 
-This will execute your content and output the proper HTML in a `_build/_page/html/<mypage>` folder.
+This will build the file as usual, and place it in an output folder called `_build/_page/html/<mypage>`.
+
 If the file is in a subdirectory relative to the `_build` folder, the HTML will be in a `_build/_page/html/<subdirectory-mypage>` folder.
 
 Your page will be called `mypage.html`.
 This will work for any [content source file](../file-types/index.md) that is supported by Jupyter Book.
 
-:::{note}
-Users should note that building **single pages** in the context of a larger project can trigger warnings and incomplete links.
-For example, building `docs/start/overview.md` will issue a number of `unknown document`, `term not in glossary`, and `undefined links` warnings.
-:::
+
+## Build an article from multiple files
+
+You may also split an article across multiple input files (e.g., if you'd like to store sections separately).
+To do so, use the `format: jb-article` option in your `_toc.yml` file.
+
+For example:
+
+```yaml
+format: jb-article
+root: index
+sections:
+- file: path/to/chapter1
```

Comment:
@choldgraf wondering if this should be

```
- file: path/to/section1
```

rather than

```
- file: path/to/chapter1
```

I think that may reduce confusion. as you can and should use `jb-book` format for a chapter based book

```
format: jb-book
root: intro
chapters:
- file: chapter1
```

Reply:
yes definitely!